styles/styles.scss

```diff
@@ -1733,4 +1729,12 @@ input[type="color"] {
       background-color: rgba(var(--bs-info-rgb), 0.25);
     }
   }
+}
+
+.text-purple {
+  color: rgb(var(--bs-purple-rgb)) !important;
+}
```

Comment:
create conflict with boostrap!

you shoud add the purple color to `variables.scss` at `$more-colors`